prac5/hex_colours.py

```diff
@@ -0,0 +1,12 @@
+CODE_TO_HEX_COLOR = {"ABSOLUTE ZERO": "#0048ba", "ACID GREEN": "#b0bf1a", "ALICEBLUE": "#f0f8ff",
+                     "ALIZARIN CRIMSON": "#e32636",
+                     "AMARANTH": "#e52b50", "AMBER": "#ffbf00", "AMETHYST": "#9966cc", "ANTIQUEWHITE": "#faebd7",
+                     "ANTIQUEWHITE1": "#ffefdb"}
+
+color_name = input("Enter color name: ")
```

Comment:
I think add a ".lower" here can be better

Reply:
There is no need to add lower